index.html

```diff
@@ -0,0 +1,63 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>Document</title>
+ <link rel="stylesheet" type="text/CSS" href="index.css"/> 
+ <link href="https://fonts.googleapis.com/css2?family=Nunito:wght@300&display=swap" rel="stylesheet">
+</head>
+<body>
+<header>   
+    <h1> A Big Girls Bedroom for a Big Girl</h1>  
+</header>
+
+<menu>
+    <a href="./main_first_project.html">Home</a>
+    <a href="./about_first_project.html">About Me</a>
+    <a href="./photogal_first_project.html">Photo Gallery</a>
+  </menu>   
```

Comment:
Nice! :) 
A suggestion is to include this menu in the other HTML-files so that the user can navigate back to the home page.